packages/docs/docs/getting-started/code-block.mdx

```diff
@@ -0,0 +1,249 @@
+---
+sidebar_position: 12
+slug: /code-block
+---
+
+import AnimationPlayer from '@site/src/components/AnimationPlayer';
+import signalSource from '!!raw-loader!@motion-canvas/examples/src/scenes/node-signal';
+
+# CodeBlock
+
+<AnimationPlayer banner name="code-block" />
+
+The `CodeBlock` component is used to present syntax highlighted code. It can
+automatically highlight many common languages, and, once displayed, that code
+can be modified in place to present changes to the viewer. Modified code will
+animate from the prior code to the new code by removing old code, translating
+the remaining code, and inserting new code. You may also "select" code in order
+to call attention to some important snippet.
+
+## Using the component
+
+To display code, set the `code` and `language` property.
+
+```ts
+import {CodeBlock} from '@motion-canvas/2d/lib/components/CodeBlock';
+
+yield view.add(
+  <CodeBlock language="c#" code={`Console.WriteLine("Hello World!")`} />,
+);
+```
+
+You can find a list of available languages in the [Starry Night][languages]
```

Comment:
Should this link to the [actual repo](https://github.com/wooorm/starry-night#languages)?